core/src/main/java/io/crate/plugin/PluginLoader.java

```diff
@@ -183,8 +183,9 @@ public String apply(Plugin input) {
                             checkJarHell(libURL);
                             pluginUrls.add(libURL);
                         } catch (Exception e) {
-                            logger.warn(
-                                    "Library " + libURL + " of plugin " + pluginURL + " already loaded, will ignore");
+                            String msg = "Library " + libURL + " of plugin " + pluginURL + " already loaded";
+                            logger.error(msg, e);
```

Comment:
while you are at it you could also use variables here like in the other logger calls